docs/source/development/reference/functions.rst

```diff
@@ -783,6 +783,17 @@ Setting custom I/O functions
 
 .. doxygenfunction:: proj_context_set_sqlite3_vfs_name
    :project: doxygen_api
+   
+.. c:function:: void proj_context_set_search_paths (PJ_CONTEXT *ctx , int count_paths, const char* const* paths)
+
+    Sets the resource files search paths.
+
+    :param ctx: Threading context.
+    :type ctx: :c:type:`PJ_CONTEXT` *
+    :param `count_paths`: Number of paths.
+    :type `count_paths`: `int`
+    :param `paths`: Array of paths.
+    :type `paths`: `const char* const*`
```

Comment:
```suggestion
    :project: doxygen_api
```

I believe that's a solution, as the function is already documented in the code https://github.com/OSGeo/PROJ/blob/5e077729274f5d28e137e1a41f7d3350146614ef/src/filemanager.cpp#L1826-L1846)